### ISSUE TITLE
Feeds disregard their inspector settings when moved between accounts.

### ISSUE THREAD

**Issue by stuartbreckenridge** [Ranchero-Software/NetNewsWire#2705]
Scenario:

- Feed is located in _On My Mac_ and has notifications enabled for new articles
- User drags feed from _On My Mac_ to _iCloud_
- The feed is migrated but notifications for new articles are not enabled.

<img width="290" alt="Screenshot 2021-01-05 at 20 50 12" src="https://user-images.githubusercontent.com/7046652/103648550-04dabf00-4f98-11eb-9abb-268825f7bf2a.png">
<img width="313" alt="Screenshot 2021-01-05 at 20 50 32" src="https://user-images.githubusercontent.com/7046652/103648558-0906dc80-4f98-11eb-8263-89ba06ebebee.png">

**Comment by vincode-io** [comment]
When you copy a feed, you only copy the URL of the feed. Basically, we don't copy modified name, starred items, unread items, feed metadata, etc...  There is a lot of complicated logic there for very little user benefit. For example, if a starred article isn't in the feed anymore, we're copying it as well now too. You could argue for copying as much as possible, but doing it right is a whole lot work.

**Comment by stuartbreckenridge** [comment]
I don't disagree. The unread articles (etc) didn't bother me as much as they are a quicker fix. The preferences take a bit more time. (Also, if you drag the feed back—in this example from iCloud back to local—the preferences are _restored_ from the point of view of the user.)

**Comment by vincode-io** [comment]
Right, that is another complication. We don't delete feed metadata so that undo will work correctly.
